language/stdlib/modules/Signature.move

```diff
@@ -23,6 +23,11 @@ module Signature {
         public_key: vector<u8>,
         message: vector<u8>
     ): bool;
+
+    spec module {
+        native define spec_ed25519_validate_pubkey(public_key: vector<u8>): bool;
```

Comment:
You can use `Signature::ed25519_validate_pubkey` and `Signature::ed25519_verify` directly in specs, since they are both pure functions and I have defined them in the prelude already. There's no need to add spec versions here and in the prelude.

Reply:
Dome removing these spec functions.